pkg/containers/containers.go

```diff
@@ -50,14 +53,28 @@ func InitContainers() (*Containers, error) {
 		}
 	}
 
+	runtimeService := RuntimeInfoService()
+
+	registerEnricher(&runtimeService, sockets, "containerd", runtime.ContainerdEnricher)
```

Comment:
error checking, also I am not sure i understand the logics here. Can you please describe it?

Reply:
Resolving since I removed the function